test/ValidatorSet/Staking.test.ts

```diff
@@ -86,12 +86,25 @@ export function RunStakingTests(): void {
       await expect(validatorSet.connect(this.signers.validators[0]).stake({ value: this.minStake })).to.not.be.reverted;
     });
 
+    it("should get all active validators", async function () {
```

Comment:
This is not a Staking function to be tested here and it doesn't return active validators